epub33/core/index.html

```diff
@@ -691,277 +732,410 @@ <h4>Conformance Checking</h4>
+										<li><code>font/ttf</code></li>
+										<li><code>application/font-sfnt</code></li>
+									</ol>
+								</td>
+								<td>[[TrueType]] </td>
+								<td>TrueType fonts</td>
+							</tr>
+							<tr>
+								<td id="cmt-otf">
+									<ol class="cmt">
+										<li><code>font/otf</code></li>
+										<li><code>application/font-sfnt</code></li>
+										<li><code>application/vnd.ms-opentype</code></li>
+									</ol>
+								</td>
+								<td>[[OpenType]]</td>
+								<td>OpenType fonts</td>
+							</tr>
+							<tr>
+								<td id="cmt-woff">
+									<ol class="cmt">
+										<li><code>font/woff</code></li>
+										<li><code>application/font-woff</code></li>
+									</ol>
+								</td>
+								<td> [[WOFF]] </td>
+								<td>WOFF fonts</td>
+							</tr>
+							<tr>
+								<td id="cmt-woff2">
+									<code>font/woff2</code>
+								</td>
+								<td> [[WOFF2]] </td>
+								<td>WOFF2 fonts</td>
+							</tr>
+							<tr>
+								<th colspan="3" id="cmt-grp-other" class="tbl-group">Other</th>
+							</tr>
+							<tr>
+								<td id="cmt-xhtml">
+									<code>application/xhtml+xml</code>
+								</td>
+								<td>
+									<a href="#sec-xhtml">XHTML Content Documents</a>
+								</td>
+								<td><a>XHTML Content Documents</a> that use the <a data-cite="html#xhtml">XHTML
+										syntax</a> [[HTML]]. </td>
+							</tr>
+							<tr>
+								<td id="cmt-js">
+									<ol class="cmt">
+										<li><code>application/javascript</code></li>
+										<li><code>application/ecmascript</code></li>
+										<li><code>text/javascript</code></li>
+									</ol>
+								</td>
+								<td> [[RFC4329]] </td>
+								<td>Scripts.</td>
+							</tr>
+							<tr>
+								<td id="cmt-ncx">
+									<code>application/x-dtbncx+xml</code>
+								</td>
+								<td> [[OPF-201]] </td>
+								<td>The <a href="#legacy">legacy</a> NCX.</td>
+							</tr>
+							<tr>
+								<td id="cmt-smil">
+									<code>application/smil+xml</code>
+								</td>
+								<td>
+									<a href="#sec-media-overlays">Media Overlays</a>
+								</td>
+								<td>EPUB Media Overlay documents</td>
+							</tr>
+						</tbody>
+					</table>
+
+					<div class="note">
+						<p>Inclusion as a Core Media Type Resource does not mean that all Reading Systems will support
+							the rendering of a resource. Reading System support also depends on the capabilities of the
+							application (e.g., a Reading System with a <a>Viewport</a> must support image Core Media
+							Type Resources, but a Reading System without a Viewport does not). Refer to <a
+								data-cite="epub-rs-33#sec-epub-rs-conf-cmt">Core Media Types</a> [[EPUB-RS-33]] for more
+							information about which Reading Systems rendering capabilities require support for which
+							Core Media Type Resources.</p>
 
 						<p>The Working Group typically only includes formats as Core Media Type Resources when they have
```

Comment:
Not sure about this one: it _may_ be better to refer to W3C not to a WG; per its nature, a WG has a time-limited existence compared to W3C...

```suggestion
						<p>W3C typically only includes formats as Core Media Type Resources when they have
```

Reply:
Do you want to take this up separately? I bet there are a lot of instances where we mention the "Working Group". One of the caution boxes even mentions what the working group thinks w3c will do...

Reply:
ok, forget about this one, and will come back to it when the current flurry of PR-s are done...